base/debian/control

```diff
@@ -2,26 +2,27 @@ Source: cuttlefish-common
 Maintainer: Cuttlefish Team
 Section: misc
 Priority: optional
-Build-Depends: config-package-dev,
-               debhelper-compat (= 12),
+Build-Depends: bazel [amd64],
                clang,
```

Comment:
curious about still needing `clang` as a dependency now that we will be using a pre-built clang binary?

Reply:
Removed.